pkgs/build-support/fetchpijul/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenvNoCC, pijul, lib }:
+
+lib.makeOverridable (
+{ url
+, sha256 ? ""
```

Comment:
This should support hash

Reply:
Ah, I was inspired by `fetchfossil` which only supports sha256, but fair enough, I'll add that